### DISCUSSION TITLE
Seeking MAUI Migration Advice

### DISCUSSION THREAD

**Discussion by KaroumMobiarchitects** [dotnet/maui#20386] [Upgrading from Xamarin]
### Description

With Microsoft ending Xamarin support, we're planning to migrate to MAUI. Seeking feedback from those who've already made the switch, particularly regarding Firebase push notifications and AppCenter integration. Any insights or tips would be greatly appreciated.

### Public API Changes

.

### Intended Use-Case

.

**Comment by drasticactions** [comment]
Thanks for trying out MAUI! As this is not an "issue" to be fixed, I am moving this to the Discussions forum. Hopefully others with insights into Firebase and AppCenter will chime in.
